src/tasks/acceptability.py

```diff
@@ -0,0 +1,433 @@
+)
+@register_task("npi_negsent_pr", rel_path="NPI/probing/sentential_negation_biclausal/npi_present")
+@register_task("npi_only_li", rel_path="NPI/probing/only/licensor")
+@register_task("npi_only_sc", rel_path="NPI/probing/only/scope_with_licensor")
+@register_task("npi_only_pr", rel_path="NPI/probing/only/npi_present")
+@register_task("npi_qnt_li", rel_path="NPI/probing/quantifiers/licensor")
+@register_task("npi_qnt_sc", rel_path="NPI/probing/quantifiers/scope_with_licensor")
+@register_task("npi_qnt_pr", rel_path="NPI/probing/quantifiers/npi_present")
+@register_task("npi_ques_li", rel_path="NPI/probing/questions/licensor")
+@register_task("npi_ques_sc", rel_path="NPI/probing/questions/scope_with_licensor")
+@register_task("npi_ques_pr", rel_path="NPI/probing/questions/npi_present")
+@register_task("npi_quessmp_li", rel_path="NPI/probing/simplequestions/licensor")
+@register_task("npi_quessmp_sc", rel_path="NPI/probing/simplequestions/scope_with_licensor")
+@register_task("npi_quessmp_pr", rel_path="NPI/probing/simplequestions/npi_present")
+@register_task("npi_sup_li", rel_path="NPI/probing/superlative/licensor")
+@register_task("npi_sup_sc", rel_path="NPI/probing/superlative/scope_with_licensor")
+@register_task("npi_sup_pr", rel_path="NPI/probing/superlative/npi_present")
+@register_task("cola_npi_adv", rel_path="NPI/splits/adverbs")
+@register_task("cola_npi_cond", rel_path="NPI/splits/conditionals")
+@register_task("cola_npi_negdet", rel_path="NPI/splits/determiner_negation_biclausal")
+@register_task("cola_npi_negsent", rel_path="NPI/splits/sentential_negation_biclausal")
+@register_task("cola_npi_only", rel_path="NPI/splits/only")
+@register_task("cola_npi_ques", rel_path="NPI/splits/questions")
+@register_task("cola_npi_quessmp", rel_path="NPI/splits/simplequestions")
+@register_task("cola_npi_qnt", rel_path="NPI/splits/quantifiers")
+@register_task("cola_npi_sup", rel_path="NPI/splits/superlative")
+@register_task("all_cola_npi", rel_path="NPI/combs/all_env")
+@register_task("wilcox_npi", rel_path="NPI/wilcox")
+@register_task("hd_cola_npi_adv", rel_path="NPI/combs/minus_adverbs")
+@register_task("hd_cola_npi_cond", rel_path="NPI/combs/minus_conditionals")
+@register_task("hd_cola_npi_negdet", rel_path="NPI/combs/minus_determiner_negation_biclausal")
+@register_task("hd_cola_npi_negsent", rel_path="NPI/combs/minus_sentential_negation_biclausal")
+@register_task("hd_cola_npi_only", rel_path="NPI/combs/minus_only")
+@register_task("hd_cola_npi_ques", rel_path="NPI/combs/minus_questions")
+@register_task("hd_cola_npi_quessmp", rel_path="NPI/combs/minus_simplequestions")
+@register_task("hd_cola_npi_qnt", rel_path="NPI/combs/minus_quantifiers")
+@register_task("hd_cola_npi_sup", rel_path="NPI/combs/minus_superlative")
+class CoLANPITask(SingleClassificationTask):
+    """Class for NPI-related task; same with Warstdadt acceptability task but outputs labels for
+       test-set
```

Comment:
Typo in Alex's name. Use full sentence.

Reply:
Ah, that's embarrassing, I should probably have noticed this when moving this code from tasks.py.